d3a_api_client/commands.py

```diff
@@ -0,0 +1,72 @@
+from d3a_api_client.enums import Commands, command_enum_to_command_name
+
+
+class ClientCommandBuffer:
+
+    def __init__(self, ):
+        self._commands_buffer = []
+
+    def offer_energy(self, area_uuid, energy, price):
+        area_uuid = area_uuid
+        callback_args = {energy, price}
+        action = Commands.OFFER
+        self.execute(area_uuid, action, callback_args)
```

Comment:
Sorry, i just saw that. IMO these are not callbac_args, but "normal" arguments for the command, so I propose renaming them to `args` or similar.
Or can you tell my you have chosen this naming?

Reply:
Yep I created this PR with a different approach and these were callback args which were not called immediately but rather later, but this was changed.
I will rename it, Thanks !